artifactory/cli.go

```diff
@@ -587,6 +594,18 @@ func GetCommands() []cli.Command {
 				return pipDepsTreeCmd(c)
 			},
 		},
+		{
+			Name:         "npm-config",
```

Comment:
I suggest placing the **npm-config** command as the first npm command (similarly to the **pip0config** command).
Let's dp the same for the **go-config** command.